packages/fluentui/react-docs/src/index.ts

```diff
@@ -0,0 +1,8 @@
+import * as docgen from './docgen';
+
+export { default as getComponentInfo } from './getComponentInfo';
+export { default as parseDefaultValue } from './parseDefaultValue';
+export { default as parseDocblock } from './parseDocblock';
+export { default as parseType } from './parseType';
```

Comment:
Not sure that this should be exported

Reply:
Slimmed down exports to just required entry points of `getComponentInfo` and `fluentuiSchema`